docs/docs-content/clusters/edge/edge-host-management/configure-proxy.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_label: "Configure HTTP-Proxy in EMC"
+title: "Configure HTTP-Proxy in EMC"
+description: "Instructions for configuring HTTP proxy in Edge Management Console."
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["edge"]
+---
+
+You can configure your Edge hosts to use an HTTP-proxy for secure internet access from Edge Management Console(EMC).
+
+## Prerequisites
+
+- An Edge host deployed with Edge Installer 4.3 or later.
+
+- An active HTTP proxy server.
```

Comment:
We might potentially mention this in a callout box, but depending on the HTTP proxy server, there may be some gotchas that are captured in this page: https://docs.spectrocloud.com/architecture/grps-proxy

Reply:
I see - is gPRC only used for communication between Palette and the cluster? If so, we may not need a callout at this moment as we don't allow a connection to Palette if the user wants to use EMC at the moment.